lib/Base/Image.cpp

```diff
@@ -22,7 +22,6 @@
 
 using namespace glow;
 
-#if WITH_PNG
```

Comment:
Maybe add a #error in the header if WITH_PNG is false.
Actually, if you have other idea on how to signal that libpng is required for this file that works too.
The thing I am after is, say I use the API for readPng and such, I would like something complaining loudly other than a linker error. In other words, if I missed that I don't have libpng and uses it, I don't want to debug the build process to find that information :).